driver/tests/swe_roe/mixed_elements.yaml

```diff
@@ -0,0 +1,64 @@
+# This test input corresponds to toy-problems/swe_roe/ex2b.c, but uses a file
```

Comment:
This isn't true, is it? Rather, it's like ex2b but using a grid with mixed elements, and the following changed file is the one that uses files for material properties and ICs, right?